Lab2/src/BLL/DI/DIConfigurator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using AutoMapper;
+using BLL.Infrastructure;
+using DAL.EntityFramework;
+using DAL.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace BLL.DI
+{
+    public static class DIConfigurator
+    {
+        public static ServiceProvider Configure(IConfigurationRoot config)
+        {
+            var businessAssembly = Assembly.Load("BLL");
+            var consoleAssembly = Assembly.Load("UI");
```

Comment:
It shouldn't be in BLL (incorrect project inheritance hierarchy)

Reply:
Fixed